tests/drivers/sensor/sbs_gauge/src/test_sbs_gauge.c

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright (c) 2021 Leica Geosystems AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+#include <drivers/sensor.h>
+
+const struct device *get_fuel_gauge_device(void)
+{
+	const struct device *dev = DEVICE_DT_GET_ANY(sbs-gauge);
+
+	if (!device_is_ready(dev)) {
+		return NULL;
+	}
```

Comment:
If the check fails the tests will dereference null pointers. Perhaps just asserting false is the right thing to do here? I don't know if ztest has setup/teardown functions.

Reply:
An assert is now used for the check.